<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sales</class>
 <widget class="QDialog" name="Sales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="finishEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>data_sprzed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>66</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>wartość      
składki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>uwagi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valuelEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>data                                
zakończenia                
ubezpieczenia</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>120</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>id_klient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>produkt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>data                                
rozpoczęcia                
ubezpieczenia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saleDateEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="beginEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addSaleBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj dane sprzedaży</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>id pracownika</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
